account_payment_return_financial_risk/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,4 @@
+Extends Account Financial Risk to manage payments returns.
+
+If any limit is exceed the partner gets forbidden to confirm sale
```

Comment:
This does not belong to this module description
As i understand, you wont be able to confirm a payment return if the limit is exceed.

Reply:
@carolinafernandez-tecnativa  the purpose of this module is to include the amount of payment returns, but it is not restricted to validating a payment return. @carlosdauden, could you clarify the scope of the module?

![image](https://github.com/user-attachments/assets/75c2341b-03d8-4dbf-b34c-83b328c9dbf6)

Reply:
Ping @carlosdauden could you please give your feedback on the above question?

Reply:
Carlos you are right, what this module does is to add a new section to the financial risk to be able to manage that amount in the same way as the rest.